packages/data-bite/src/CdcDataBite.tsx

```diff
@@ -206,11 +281,17 @@ const CdcDataBite = (
 
     let numericalData = []
 
-    //Get the column's data
-    filteredData.forEach(row => {
-      let value = numberFromString(row[dataColumn])
-      if(typeof value === 'number') numericalData.push(value)
-    });
+
+   // Get the column's data
+   // TODO: this  conde line probably will not work properly. Need to be tested
```

Comment:
@Atash3000 - Thanks for calling this out. I tested filters on my local and they appear to be working. You can safely remove this comment.